sdk/mixedreality/Azure.MixedReality.RemoteRendering/README.md

```diff
@@ -0,0 +1,386 @@
+# Azure Remote Rendering client library for .NET
+
+Azure Remote Rendering (ARR) is a service that enables you to render high-quality, interactive 3D content in the cloud and stream it in real time to devices, such as the HoloLens 2.
+
+This SDK offers functionality to convert assets to the format expected by the runtime, and also to manage
+the lifetime of remote rendering sessions.
+
+> [!Note] Once a session is running, a client application will connect to it using one of the "runtime SDKs".
```

Comment:
nit: Github doesn't render the `[!Note]` syntax in any way. Might be worth using **NOTE:** or something similar.